### PR TITLE
Readme for VEX Flag use cases

### PR DESCRIPTION
**Pull request by sparrell**
This sets the stage for the VEX flag use cases

### CODE REVIEW

**Comment by davaya** [comment]
As part of the process check, the README includes a note to include a link to the Healthcare PoC.  Reviewers could:
1) create another PR to add the link, or
2) use the GitHub process to suggest that the PR submitter add the link: https://www.ntia.doc.gov/files/ntia/publications/ntia_sbom_healthcare_update-2021-04-29.pdf,
perhaps with additional information https://healthitsecurity.com/features/using-software-bill-of-materials-sboms-for-medical-device-security

#2 is by far the lower overhead approach, so please consider this a review comment.

**Comment by davaya** [comment]
My only other review comment is that VEX is described as a profile of [CSAF ](https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/prose/csaf-v2-editor-draft.md) (see also https://ntia.gov/files/ntia/publications/vex_one-page_summary.pdf), The relationship between SBOM documents and VEX flags is not clear; it would be helpful to include a reference to CSAF documents as a source of VEX flag values.

dpk review:  Approve this PR, work on CSAF text for a future PR.

File: docs/Pace_Sbom_Vex_Flags_Prioritization/README.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+into the decision making engine.
+
+## Industry/Organization
+Which playbook/actions are executed will also depend on the
+policies of the organization.
+Different organizations will have different
+budgets, expertise, technologies, and priorities.
+And these will be informed/influenced by the industry they are in.
+The industries/organizations for this set of use cases are:
+A healthcare
```
Comment (dlemire60) [inline docs/Pace_Sbom_Vex_Flags_Prioritization/README.md]: I believe the intended formatting of this was a 2-level list, but that formatting isn't apparent when this README is viewed as "rich diff". I believe the markdown needs tuning to convey the intended structure.

File: docs/Pace_Sbom_Vex_Flags_Prioritization/README.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+   + not_present (i.e. there will be use cases for when there is no SBOM for that device)
+- format/serialization/version
+   + format = CycloneDX
+      - serialization = xml or json (distinction will not matter for these use cases at this time)
+   + format = SPDX
+      - serialization = xml, json, ... (distinction will not matter for these use cases at this time)
+   + For the purpose of these use cases at the moment, version of the format or serialization is being ignored  - but it is a parameter to be used by PACE
+- version
+   + for the purpose of these use cases at the moment, version of the SBOM is being ignored - but it is a parameter to be used by PACE
+- note that for a given device, PACE PAR may contain more than one SBOM, of more than one format.
```
Comment (dlemire60) [inline docs/Pace_Sbom_Vex_Flags_Prioritization/README.md:220]: Don't know what "PAR" is in "PACE PAR". Please expand.

Comment (sparrell) [reply docs/Pace_Sbom_Vex_Flags_Prioritization/README.md:220]: @dlemire60 - I'm not sure the use cases should be tutorial. Maybe they should refer to other docs to understand SBOMs or VEXs or PACE. I am ok with expanding acronyms. I got Posture Collection System, Posture Evaluation System and Posture Attribute Repository from other PACE docs. If I should be using different, let me know.

**Comment by dlemire60** [comment]
I'm reading the README file on @sparrell's fork / vex0 branch for simplicity / clarity. Notes:

- The info on the use case naming needs to appear earlier, around "VEX Flag Distinctions", as currently the names are used before they are explained
- In the SBOM section I'm very confused by Completeness: "contains known-unknowns (i.e. specific denoted known-unknowns)"; what does it mean for an unknown to be "specific denoted / delineated"? (latter term used in next bullet)

Neither of which is a reason not to merge this material into the PACE repo, so 👍 from me.

**Pull request merged**